.github/workflows/ci.yml

```diff
@@ -0,0 +1,98 @@
+name: Alfresco REST API Explorer CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - fix/**
+      - feature/**
+  push:
+    branches:
+      - master
+      - fix/**
+      - feature/**
+  workflow_dispatch:
+  schedule:
+    - cron: '0 5 * * 3'
+
+env:
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  RELEASE_VERSION: "7.3.0"
+  DEVELOPMENT_VERSION: "7.4.0-A1-SNAPSHOT"
+  MAVEN_CLI_OPTS: "-B -q -e -fae -V -DinstallAtEnd=true -U"
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 30
+
+jobs:
+  pre_commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.34.2
+
+  build:
+    name: "Build + deploy"
+    runs-on: ubuntu-latest
+    needs: [pre_commit]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.34.2
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.34.2
+      - name: "Init"
+        run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+      - name: "Build"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: mvn ${MAVEN_CLI_OPTS} verify -Dlogging.root.level=off -Dspring.main.banner-mode=off
+
+  tests:
+    name: "Integration tests"
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: "!contains(github.event.head_commit.message, '[skip tests]')"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.34.2
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.34.2
+      - name: "Init"
+        run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+      - name: "Run tests"
+        run: bash CI/test.sh
```

Comment:
Can we get rid of this script? It looks to be a one-liner that just executes `mvn verify` from what I can see.

Reply:
Done.